components/Tabs/src/MDCTabBarViewController.h

```diff
@@ -71,6 +71,9 @@ IB_DESIGNABLE
  If you provide this method, you can control whether tapping on a tab bar item actually
  switches to that viewController. If not provided, MDCTabBarViewController will always switch.
 
+ The tab bar controller calls this method regardless of whether the selected view controller
+ changed.
```

Comment:
I think I misread this comment when I first went through it.  If I understand this PR correctly, I think the following phrasing might be clearer to readers.  Please feel free to edit. :)

> @note The tab bar controller will call this method even when the tapped tab bar item is the currently-selected tab bar item.

Reply:
I pulled this text directly from UITabBarController's docs, but I think your suggestion is clearer.  I'll update this PR in just a moment.